packages/shared/src/dto/workflows/workflow-commons-fields.ts

```diff
@@ -5,9 +5,23 @@ import { WorkflowResponseDto } from './workflow-response-dto';
 import { Slug, StepTypeEnum, WorkflowPreferences } from '../../types';
 
 export type IdentifierOrInternalId = string;
+export enum UiControlGroupEnum {
+  INBOX = 'INBOX',
+}
+export class UiSchema {
+  controlGroup?: UiControlGroupEnum.INBOX;
+  elements?: UiElement[];
+}
+class UiElement {
+  elementTypeOverride: UiElementTypeEnum;
+}
 
+enum UiElementTypeEnum {
+  EMAIL_EDITOR = 'EMAIL_EDITOR',
+}
```

Comment:
Should all of these be deleted with the new approach?
```suggestion
```

Reply:
i will, i thought to delete it separately while removing the controls from the /workflows response